### ISSUE TITLE
Add duplicati to your comparison matrix.

### ISSUE THREAD

**Issue by ovizii** [gilbertchen/duplicacy#19]
I would be very interested to see how it stacks up https://github.com/duplicati/duplicati/releases

**Comment by gilbertchen** [comment]
Duplicati is a C# re-implementation of duplicity so these two should be similar.

**Comment by ovizii** [comment]
Thanks!

**Comment by CodesInChaos** [comment]
@gilbertchen That's only true for duplicati 1.x. Duplicati 2.x uses a hash based deduplication approach, though using fixed sized blocks.

http://www.duplicati.com/articles/Storage-Engine/

**Comment by gene1wood** [comment]
@gilbertchen Indeed, I'm not sure if this is still true now with Duplicati 2.x. I found [this comparison](https://forum.duplicati.com/t/duplicati-2-vs-duplicacy-2/592) of Duplicati 2 and Duplicacy. I'm unsure if I should open a new issue suggesting Duplicati 2 be added to the comparison matrix or not.

**Comment by gilbertchen** [comment]
I hope someone who is familiar with Duplicati can update the scripts to include Duplicati and post the results here.  I really don't have time to learn how to use Duplicati CLI and avoid beginner mistakes...

**Comment by DonaldTsang** [comment]
@gilbertchen same, because most people recommend duplicity (see https://www.gigenet.com/blog/linux-encryption-backup-tools/) and I would like to see comparison of that with similar named tools like duplicacy and duplicati.

**Comment by DonaldTsang** [comment]
@ovizii could you re-open this so that duplicacy can test it out?
